### ISSUE TITLE
Issue with Xcode 13 beta 3

### ISSUE THREAD

**Issue by Aurther-Nadeem** [joogps/SlideOverCard#8]
```Manager.swift

UIApplication.shared.windows.first?.rootViewController?.present(controller, animated: false)
        DispatchQueue.main.asyncAfter(deadline: .now()+0.1) {

```

'shared' is unavailable in application extensions for iOS: Use view controller based solutions where appropriate instead.

**Comment by Aurther-Nadeem** [comment]
Manager.swift

**Comment by Aurther-Nadeem** [comment]
Fixed with PR #7 waiting to Merge
